Lombiq.HelpfulLibraries.OrchardCore/Mvc/TypedRoute.cs

```diff
@@ -27,35 +27,32 @@ public class TypedRoute
     private readonly string _area;
     private readonly Type _controller;
     private readonly MethodInfo _action;
-    private readonly List<KeyValuePair<string, string>> _arguments;
+    private readonly IList<KeyValuePair<string, string>> _arguments;
```

Comment:
```suggestion
    private readonly List<KeyValuePair<string, string>> _arguments;
```

Reply:
I wish I could make it IReadOnlyList because outside the constructor this collection is read only.

Reply:
Why can't you do that, then?

Reply:
🤷‍♀️

Reply:
No, really? :D

Reply:
Yeah really: https://github.com/Lombiq/Helpful-Libraries/pull/235/commits/15176e3822114ca482918fb58ec72fa1ad1ab3c3

Reply:
It's possibru :O.